jurigged/live.py

```diff
@@ -98,7 +107,11 @@ def on_prepare(self, module_name, filename):
     def refresh(self, path):
         cf = self.registry.get(path)
         try:
+            if self.prerun_callback:
+                self.prerun_callback()
```

Comment:
I would add `path` and `cf` as arguments to the callback, to increase the range of useful things that could be done with this.